lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskServiceImpl.java

```diff
@@ -1107,13 +1107,22 @@ private BulkOperationResults<String, TaskanaException> completeTasks(
                 addErrorToBulkLog(this::checkPreconditionsForCompleteTask, bulkLog));
       } else {
         String userId = taskanaEngine.getEngine().getCurrentUserContext().getUserid();
+        User user = userMapper.findById(userId);
+        String userLongName;
+        if (user != null) {
+          userLongName = user.getLongName();
+        } else {
+          userLongName = null;
```

Comment:
I think this is unnecessary

Reply:
If I try to do it differently, the compiler will either say: "Variable 'userLongName' might not have been initialized" or "Variable used in lambda expression should be final or effectively final" when calling claimActionsOnTask later. How can it be avoided?

Reply:
You could initialize userLongName with null in line 1110. Then you don't need the else parts anymore